WordPress/Classes/ViewRelated/Blog/Blogging Reminders/Time Selector/Date+BloggingReminders.swift

```diff
@@ -0,0 +1,16 @@
+import Foundation
+
+extension Date {
```

Comment:
Since this is used in the `TimeSelectionView` which is decoupled from the Blogging reminders, I'm wondering if this needs to have the `+BloggingReminders` in the name, could we make this more generic? 

Or make it private to blogging reminders so others don't accidentally use it.

Reply:
That's right, I'll make it more generic